app/controllers/solid_errors/errors_controller.rb

```diff
@@ -2,20 +2,16 @@ module SolidErrors
   class ErrorsController < ApplicationController
     around_action :force_english_locale!
 
-    before_action :set_error, only: %i[show update]
+    before_action :set_error, only: %i[show update destroy]
 
     # GET /errors
     def index
-      errors_table = Error.arel_table
-      occurrences_table = Occurrence.arel_table
+      @errors = get_errors(base_scope: Error.unresolved)
+    end
 
-      @errors = Error.unresolved
-        .joins(:occurrences)
-        .select(errors_table[Arel.star],
-          occurrences_table[:created_at].maximum.as("recent_occurrence"),
-          occurrences_table[:id].count.as("occurrences_count"))
-        .group(errors_table[:id])
-        .order(recent_occurrence: :desc)
+    # GET /errors/resolved
+    def resolved
+      @errors = get_errors(base_scope: Error.resolved)
```

Comment:
I would prefer using a query param for the scope, which has a default. Something like:
```ruby
def index
  errors_table = Error.arel_table
  occurrences_table = Occurrence.arel_table
  scope = params[:scope] == "resolved" ? Error.resolved : Error.unresolved
  @errors = scope
    .joins(:occurrences)
    .select(errors_table[Arel.star],
      occurrences_table[:created_at].maximum.as("recent_occurrence"),
      occurrences_table[:id].count.as("occurrences_count"))
    .group(errors_table[:id])
    .order(recent_occurrence: :desc)
end

Reply:
I don't like non-RESTful actions, plus it will make the path helpers easier